pss-migration-orchestration/README.md

```diff
@@ -0,0 +1,86 @@
+
+## :beginner: Current state / context
+
+PSP has been deprecated and replaced in Giant Swarm clusters with Kyverno PSS policies.
+
+This change means customers must change the way they handle exceptions to policies for resources which are not compliant.
+
+This effort is highly variable depending on the way the organization manages exceptions and the current state of compliance.
+
+We intend to introduce a new CRD and build two new controllers, based on our larger policy orchestration concept, but initially with highly migration-specific logic, to automate most of the toil for PSS migration.
+
+## :triangular_flag_on_post: Problem space
+
+For various reasons, customers may find that the move to PSS involves significant effort.
+
+- At minimum, resources which were permitted under PSP will need to receive Kyverno exceptions if they are to continue running
+  - Platform teams or workload owners will need to create a new exception per failing resource (or, though not ideal, create more broadly-scoped exceptions or disable policies entirely)
+  - There is a slight learning curve to creating Kyverno PolicyExceptions
+  - With many workloads requiring exceptions, this is either a large task done centrally or smaller tasks coordinated across many teams
+- Per organizational policy or implementation reasons, teams may need to re-apply (e.g. for security approval) for previously approved exceptions
+
+## 📈  Solution space
+
+Some approaches to ease the PSS transition are:
+
+- Documentation. We have produced multiple rounds of instructional PSS-related content, but this does not ultimately reduce the effort of actually applying those instructions
+- Dashboards / reports. In addition to the native reports, we operate global dashboards for as much of the adoption status as possible, and have deployed a Kyverno UI per-cluster for use in identifying failing workloads. However, these also do not eliminate the manual touches needed for customers to actually assume management of the new exceptions.
+- Generation of exceptions from PSPs. This approach was attempted with partial success. However, the policies are not a perfect 1-1 match, and there are policies in PSS which do not have direct equivalents in PSP, so there remains some ambiguity and customer decision required for their enforcement.
+- Generate Kyverno exceptions from PolicyReports. This has also been trialed, with success. It is possible to simply generate an exception satisfying every failing policy for a given workload. However, this introduces two problems. First, without some review, it potentially grants more permissions than intended by the cluster admins. Additionally, since there is no backing source for the PolicyException, the workload will abruptly stop working at some point after its exception is removed. The PolicyExceptions must be properly managed in a deployed resource which will re-apply them in the future.
```

Comment:
Maybe we can tag somehow automatically generated policies and then build a dashboard/alerting allowing customers to act?

Reply:
AN alterantive would be to generate the exception, but in a way that the customer can download and manage it in any way they want (centrally or with the app itself)